tests/bouncer/views_test.py

```diff
@@ -162,6 +162,15 @@ def test_annotation_strips_bare_fragment_identifiers(self, parse_document):
         assert data["viaUrl"] == (
                 "https://via.hypothes.is/http://example.com/example.html#annotations:AVLlVTs1f9G3pW-EYc6q")
 
+    def test_annotation_omits_via_url_for_third_party_annotations(self, parse_document):
+        parse_document.return_value["authority"] = "partner.org"
+        template_data = (
```

Comment:
The ah ... ostensible reason is "consistency with other code in the same module". The actual reason is I copied and pasted these lines of code from another test.

I've compacted this and the other occurrences in a separate commit for consistency, but it would probably be worthwhile to factor them into a helper method in future.